test/bigtest/network/factories/item.js

```diff
@@ -14,11 +15,14 @@ export default Factory.extend({
   missingPieces: () => '',
   numberOfPieces: () => 1,
 
-  withLoan: trait({
-    afterCreate(item, server) {
-      server.create('loan', {
-        item
-      });
-    }
-  })
+  withLoan: {
+    extension: {
+      afterCreate(item, server) {
+        server.create('loan', {
+          item
+        });
+      }
+    },
+    __isTrait__: true
```

Comment:
`__isTrait__` looks suspiciously like mucking about with an internal API, though I confess I'm not very familiar with mirage. Do you think there is reason for concern here, or perhaps a different/better way to achieve this behavior without `__isTrait__`?

Reply:
Oops, that was an oversight on my end. It looks like `miragejs` offers `trait()` out of the box. I'll go ahead and update all the other pull requests as well. Sorry about that. 👍